DQM/SiStripCommissioningAnalysis/interface/SiStripPulseShape.h

```diff
@@ -21,52 +19,45 @@
 #define SiStripPulseShape_h_
 
 double fpeak(double *x, double *par);
-
 double fdeconv(double *x, double *par);
 
-double fpeak_convoluted(double *x, double *par);
+double fturnOn(double *x, double *par);
+double fdecay(double *x, double *par);
 
+double fpeak_convoluted(double *x, double *par);
 double fdeconv_convoluted(double *x, double *par);
 
 double pulse_raw(double x, double y, double z, double t);
-
 double pulse_x0(double y, double z, double t);
-
 double pulse_yz(double x, double z, double t);
-
 double pulse_x0_yz(double z, double t);
-
 double pulse(double x, double y, double z, double t);
 
-double get_compensation(double x);
-
-
-class SiStripPulseShape
-{
-  public: 
-    enum mode {peak,deconvolution};
-    SiStripPulseShape():mode_(deconvolution) {}
-    virtual ~SiStripPulseShape() {}
-    inline void setMode(const mode theMode) { mode_=theMode; }
-    inline mode getMode() const { return mode_; } 
-    inline double getNormalizedValue(const double& t) const
-    {
-      double parameters[5]={0.,-2.82,0.066,50,20};
-      double time = t;
-      switch(mode_) {
-       case peak:
-        {
-	  return fpeak(&time,parameters);
-	}
-       case deconvolution:
-        {
-	  return fdeconv(&time,parameters);
-	}
+class SiStripPulseShape{
+ public: 
+  enum mode {peak,deconvolution};
+ SiStripPulseShape():mode_(deconvolution) {}
+  virtual ~SiStripPulseShape() {}
+  inline void setMode(const mode theMode) { mode_=theMode; }
+  inline mode getMode() const { return mode_; } 
+  inline double getNormalizedValue(const double& t) const
+  {
+    double parameters[5]={0.,-2.82,0.066,50,20};
+    double time = t;
+    switch(mode_) {
+    case peak:
+      {
+	return fpeak(&time,parameters);
+      }
+    case deconvolution:
+      {
+	return fdeconv(&time,parameters);
```

Comment:
Could you please take a look at the formatting of this newly introduced block of code?

Reply:
@andrius-k: I am sorry but I haven't understood your request, what do you mean exactly? The SiStripPulseShape class was already defined in earlier versions of the code. See for example[0]. I haven't touched it recently.

[0] https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/DQM/SiStripCommissioningAnalysis/interface/SiStripPulseShape.h#L28-L55

Reply:
@rgerosa If you have not changed the code, then please do not change the formatting either, at least not for the worse.

Reply:
@schneiml: thanks for the feedback ... therefore I am just going update the pull request taking out the std::cout that were just used for testing reasons (placeholders that I forgot to take out).

Reply:
@rgerosa The formatting of the code is different in your PR vs the previous version (the one that you linked). Please take a look at the diff above this thread. The formatting was not exactly conforming to standards to begin with, so if you want to change it for the better you're really welcome to do so!

Reply:
@andrius-k sorry for the silly question but I am personally not aware of guidelines for code formatting in CMSSW. In order to update the code accordingly, where can I them? This would be very useful in order to avoid mistakes in future. Thanks in advance!

Reply:
@rgerosa You can find the CMSSW style guide here: http://cms-sw.github.io/cms_coding_rules.html
The instructions are kind of brief and don't cover all of the cases so in general, following the rule of "when editing, use the previous style" is ok if it makes the style inside a file or group of files consistent. 
Not trying to nitpick here, just noticed that a whitespace of the block of code became slightly less consistent while the code itself didn't change.

Reply:
@andrius-k: I have just updated the PR following your suggestions. This time the code style should be more consistent.